source/sgp_mode/Tasks.h

```diff
@@ -144,6 +148,35 @@ class SquareTask : public OutputTask {
     OutputTask::MarkPerformed(state, output, task_id, shared);
     state.internalEnvironment->insert(state.internalEnvironment->begin(),
                                       sqrt(output));
+    if(state.host->IsHost()){
+      IncrementSquareMap(output, 1);
```

Comment:
Since the parameter is boolean type, you might as well pass true and false instead of 1 and 0 to make that clearer